src/app/ui/copy-to-clipboard.js

```diff
@@ -16,9 +14,6 @@ var css = csjs`
 
 module.exports = function copyToClipboard (getContent, tip = 'Copy value to clipboard', icon = 'fa-clipboard') {
   var copyIcon = yo`<i title="${tip}" class="${css.copyIcon} fa ${icon}" aria-hidden="true"></i>`
-  copyIcon.style.color = styles.remix.icon_Color_CopyToClipboard
-  copyIcon.onmouseenter = function (event) { copyIcon.style.color = styles.remix.icon_HoverColor_CopyToClipboard }
```

Comment:
so for the moment the icon won't have a hover color.

Reply:
yes let's tackle that afterward. i did not find a good solution for this